pkg/daemon/discover/discover.go

```diff
@@ -32,6 +32,7 @@ import (
 
 	"github.com/coreos/pkg/capnslog"
 	"github.com/rook/rook/pkg/clusterd"
+	cephController "github.com/rook/rook/pkg/operator/ceph/controller"
```

Comment:
```suggestion
	cephcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
```

I'd suggest keeping the same pattern with have today, we don't use this format anywhere in the Rook codebase.

Reply:
Also, squash your commits.

Reply:
I will squash my commit, however https://github.com/rook/rook/blob/8c8b74afc3c8a5dc6150d6b535b01483e5816c39/pkg/operator/ceph/operator.go#L34 uses cephController and I can't any anywhere in the Rook codebase uses `cephcontroller`

Reply:
https://github.com/rook/rook/blob/1c1591c8478478cd93c8e99fdeb8098dd68fa231/pkg/operator/ceph/csi/spec.go#L26

Reply:
So should I change `rook/pkg/operator/ceph/operator.go` as well?

Reply:
I don't see it in this file. I am confused.

Reply:
I'll use `cephcontroller` as you suggested, FYI here are current naming in codebase

```
pkg/operator/ceph/operator.go:	cephController "github.com/rook/rook/pkg/operator/ceph/controller"
pkg/operator/ceph/csi/spec.go:	controllerutil "github.com/rook/rook/pkg/operator/ceph/controller"
pkg/operator/ceph/file/controller.go:	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
pkg/operator/ceph/nfs/controller.go:	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
pkg/operator/ceph/pool/controller.go:	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
pkg/operator/ceph/object/rgw.go:	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
pkg/operator/ceph/object/controller.go:	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
pkg/operator/ceph/object/user/controller.go:	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
```